crates/viewer/re_view_spatial/tests/transform_hierarchy.rs

```diff
@@ -216,7 +258,7 @@ fn run_view_ui_and_save_snapshot(
             );
             test_context.handle_system_commands(&harness.ctx);
 
-            harness.run();
+            harness.run_steps(10);
```

Comment:
Not really sure why the above changes would make this necessary 🤔

Reply:
Wow, good catch! I was running into some infinite loop errors and forgot to add a TODO. Still need to figure that out.